spec/components/provider_interface/application_card_component_spec.rb

```diff
@@ -109,4 +130,78 @@
       end
     end
   end
+
+  describe '#contextual_date' do
+    let(:reject_by_default_at) { DateTime.parse('2020-06-02T09:05:00+01:00') }
+    let(:updated_at) { DateTime.parse('2020-06-02T09:05:00+01:00') }
+    let(:status) { 'awaiting_provider_decision' }
+    let(:show_date) { :reject_by_default_at }
+    let(:application_choice) do
+      build_stubbed(
+        :application_choice,
+        reject_by_default_at: reject_by_default_at,
+        updated_at: updated_at,
+        course_option: course_option,
+        status: status,
+      )
+    end
+
+    subject(:component) { described_class.new(application_choice: application_choice, show_date: show_date) }
+
+    context 'when not sorting by reject by default date' do
+      let(:show_date) { :updated_at }
+
+      it 'presents the last changed date' do
+        expect(component.contextual_date).to eq('Changed  2 June 2020 at  9:05am')
+      end
+    end
+
+    context 'when application is not in awaiting_provider_decision state' do
+      let(:status) { 'withdrawn' }
+
+      it 'presents the last changed date' do
+        expect(component.contextual_date).to eq('Changed  2 June 2020 at  9:05am')
+      end
+    end
+
+    context 'when reject_by_default_at is nil' do
+      let(:reject_by_default_at) { nil }
+
+      it 'presents the last changed date' do
+        expect(component.contextual_date).to eq('Changed  2 June 2020 at  9:05am')
+      end
+    end
+
+    context 'when reject_by_default_at is in the past' do
+      let(:reject_by_default_at) { 1.day.ago }
+
+      it 'presents the last changed date' do
+        expect(component.contextual_date).to eq('Changed  2 June 2020 at  9:05am')
+      end
+    end
+
+    context 'when reject_by_default_at is less than a day away' do
+      let(:reject_by_default_at) { 1.hour.from_now }
+
+      it 'presents "Less than 1 day to respond"' do
+        expect(described_class.new(application_choice: application_choice, show_date: show_date).contextual_date).to eq('Less than 1 day to respond')
```

Comment:
I found these still passed relying on the subject. With `contextual_date` in the subject, this could be

```ruby
      it 'presents "Less than 1 day to respond"' do
        expect(contextual_date).to eq('Less than 1 day to respond')
      end
```

or even

```ruby
it { is_expected.to eq 'Less than 1 day to respond' }
```